react-intl/index.d.ts

```diff
@@ -227,7 +228,11 @@ declare namespace ReactIntl {
             defaultFormats?: Object;
         }
     }
-    class IntlProvider extends React.Component<IntlProvider.Props, any> { }
+    class IntlProvider extends React.Component<IntlProvider.Props, any> {
+        getChildContext(): {
+            intl: any;
```

Comment:
It includes `InjectedIntl` with [a load of other stuff](https://github.com/yahoo/react-intl/blob/master/src/components/provider.js#L147). It looks something like this:
```
{ 
      locale: 'en',
      formats: {},
      messages: {},
      defaultLocale: 'en',
      defaultFormats: {},
      formatDate: [Function: bound formatDate],
      formatTime: [Function: bound formatTime],
      formatRelative: [Function: bound formatRelative],
      formatNumber: [Function: bound formatNumber],
      formatPlural: [Function: bound formatPlural],
      formatMessage: [Function: bound formatMessage],
      formatHTMLMessage: [Function: bound formatHTMLMessage],
      formatters:
       { getDateTimeFormat: [Function],
         getNumberFormat: [Function],
         getMessageFormat: [Function],
         getRelativeFormat: [Function],
         getPluralFormat: [Function] },
      now: [Function: now] 
}
```

Given it's likely to be used in a test context, I felt it was acceptable to put an `any` in